src/exberry_int/integration_exberry.py

```diff
@@ -62,15 +63,18 @@ def integration_exberry_main(
     env: 'ExberryIntegrationEnv',
     events: 'IntegrationEvents'):
 
-    outbound_queue = asyncio.Queue()
+    outbound_queue = asyncio.PriorityQueue()
     session_started = asyncio.Event()
 
-    LOG.info(f"Preparing session...")
-    outbound_queue.put_nowait(make_order_book_depth())
-
-    async def enqueue_outbound(msg: dict):
+    async def enqueue_outbound(msg: dict, priority: int = 1):
+        global CURRENT_PRIORITY
         LOG.info(f"Enqueuing outbound message: {msg}")
-        await outbound_queue.put(msg)
+        await outbound_queue.put((priority, msg))
```

Comment:
It's odd to me that the `asyncio` uses a tuple for this... seems wasteful. (But that's what it is... :-) )

Reply:
Yep, agreed